packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -93,13 +93,18 @@ product-no-such-plan = No such plan for this product.
 
 ## payment legal blurb
 payment-legal-copy-stripe-and-paypal-2 = { -brand-name-mozilla } uses { -brand-name-stripe } and { -brand-name-paypal } for secure payment processing.
-payment-legal-link-stripe-and-paypal-2 = View the <stripePrivacyLink>{ -brand-name-stripe } privacy policy</stripePrivacyLink> and <paypalPrivacyLink>{ -brand-name-paypal } privacy policy</paypalPrivacyLink>.
+payment-legal-link-stripe-and-paypal-2 = <stripePrivacyLink>{ -brand-name-stripe } privacy policy</stripePrivacyLink> | <paypalPrivacyLink>{ -brand-name-paypal } privacy policy</paypalPrivacyLink>.
 
 payment-legal-copy-paypal = { -brand-name-mozilla } uses { -brand-name-paypal } for secure payment processing.
-payment-legal-link-paypal = View the <paypalPrivacyLink>{ -brand-name-paypal } privacy policy</paypalPrivacyLink>.
+payment-legal-link-paypal = <paypalPrivacyLink>{ -brand-name-paypal } privacy policy</paypalPrivacyLink>.
 
 payment-legal-copy-stripe-2 = { -brand-name-mozilla } uses { -brand-name-stripe } for secure payment processing.
-payment-legal-link-stripe-2 = View the <stripePrivacyLink>{ -brand-name-stripe } privacy policy</stripePrivacyLink>.
+payment-legal-link-stripe-2 = <stripePrivacyLink>{ -brand-name-stripe } privacy policy</stripePrivacyLink>.
```

Comment:
These localization IDs need to be changed for the string changes to propagate in other languages.